articles/remote-yara-rules.md

```diff
@@ -0,0 +1,91 @@
+# Remote deployment of YARA rules
+
+Fleetd with osquery can scan files on macOS and Linux systems using
```

Comment:
The YARA tables work on Windows too. Right?

Reply:
Yes great catch! I was thinking of `yara_events` which does not work on Windows.